### PR TITLE
Sketch proof missing case

### PR DESCRIPTION
**Pull request by madeve-unipi**
Sketched the proof of the missing case for the three lines lemma, with several "sorry"s.
This should hopefully build now.

### CODE REVIEW

File: BonnAnalysis/ComplexInterpolation.lean [hunk 1]
```diff
@@ -94,6 +97,77 @@ def at_height (f:ℂ → ℂ) (y:ℝ) : (Icc 0 1 : Set ℝ) → ℝ  := fun x 
...
+    simp
+  }
+  rw[this, Homeomorph.isPreconnected_preimage Complex.equivRealProdCLM.toHomeomorph]
+  exact IsPreconnected.prod isPreconnected_Ioo isPreconnected_univ
+}
+
+lemma isOpen_strip : IsOpen { z : ℂ | z.re ∈ Ioo 0 1} := by{
+  rw[strip_prod]
+  exact IsOpen.reProdIm isOpen_Ioo isOpen_univ
+}
```
Comment (pitmonticone) [inline BonnAnalysis/ComplexInterpolation.lean:158]: If you want, you can turn it into a proof term as follows (tested: can be committed directly): 


```suggestion
lemma isOpen_strip : IsOpen { z : ℂ | z.re ∈ Ioo 0 1} :=
  strip_prod ▸ IsOpen.reProdIm isOpen_Ioo isOpen_univ
```

File: BonnAnalysis/ComplexInterpolation.lean [hunk 2]
```diff
@@ -94,6 +97,77 @@ def at_height (f:ℂ → ℂ) (y:ℝ) : (Icc 0 1 : Set ℝ) → ℝ  := fun x 
...
+
+lemma isClosed_clstrip : IsClosed { z : ℂ | z.re ∈ Icc 0 1} := by{
+  rw[clstrip_prod]
+  exact IsClosed.reProdIm isClosed_Icc isClosed_univ
+}
```
Comment (pitmonticone) [inline BonnAnalysis/ComplexInterpolation.lean:163]: Same here (tested): 

```suggestion
lemma isClosed_clstrip : IsClosed { z : ℂ | z.re ∈ Icc 0 1} :=
  clstrip_prod ▸ IsClosed.reProdIm isClosed_Icc isClosed_univ
```

File: BonnAnalysis/ComplexInterpolation.lean [hunk 3]
```diff
@@ -25,7 +28,7 @@ variable {α β E E₁ E₂ E₃ : Type*} {m : MeasurableSpace α} {n : Measurab
 /- TODO: split proofs into smaller lemmas to recycle code -/
 
 -- All these names are probably very bad
-lemma Real.rpow_le_rpow_iff_left {M:ℝ} (hM: M>0) (a b : ℝ) : M^a ≤ M^b ↔ ((1 ≤ M ∧ a ≤ b ) ∨ (M ≤ 1 ∧ b ≤ a)) := by{
+lemma Real.pow_le_pow_iff {M:ℝ} (hM: M>0) (a b : ℝ) : M^a ≤ M^b ↔ ((1 ≤ M ∧ a ≤ b ) ∨ (M ≤ 1 ∧ b ≤ a)) := by{
```
Comment (fpvandoorn) [inline BonnAnalysis/ComplexInterpolation.lean:31]: Please undo this name change. I tried to improve the name in https://github.com/fpvandoorn/BonnAnalysis/commit/3a797ef90cd5905b4ff95cd28bc2c6b242a201b0

File: BonnAnalysis/ComplexInterpolation.lean (hunk 1 above)
Comment (fpvandoorn) [reply BonnAnalysis/ComplexInterpolation.lean:158]: Both proof styles are perfectly fine. I personally also prefer writing `rw + exact`.

File: BonnAnalysis/ComplexInterpolation.lean (hunk 3 above)
Comment (madeve-unipi) [reply BonnAnalysis/ComplexInterpolation.lean:31]: Thanks! I had seen that, but we also changed a hypothesis in a theorem on Friday and I was waiting to fix the other statements that used that theorem before recommitting. I'm not sure what order I am supposed to do things now. Is it fine if I just change this manually in my local version and recommit within this PR?

File: BonnAnalysis/ComplexInterpolation.lean (hunk 1 above)
Comment (madeve-unipi) [reply BonnAnalysis/ComplexInterpolation.lean:158]: Thanks for the comment, I did not know (or possibly forgot) you could do that

File: BonnAnalysis/ComplexInterpolation.lean (hunk 3 above)
Comment (fpvandoorn) [reply BonnAnalysis/ComplexInterpolation.lean:31]: Yes, that works. If you push to this branch, this PR will be automatically updated.
(If you only want to commit part of your changes, you can do that on the command line using `git add -p <myFile>` and then afterwards `git commit -m "my message"` (without the `-a` flag).

**Comment by madeve-unipi** [comment]
I had a local merge conflict for some reason (I think @pitmonticone also modified the name of the same lemma in yet another way at some point) so I may have messed up my commit. I don't want to upload Dual_private to master.
I also hope nothing broke in ComplexInterpolation along the way. Every other time I internally solved a merge conflict, the committed file got duplicate code that originally wasn't in my editor.

**Comment by pitmonticone** [comment]
Hi @madeve-unipi, Idon’t believe I have made any significant changes. The only modification was resolving the previous merge conflict by substituting the incoming change (your commit) in place of the current change (previous state).

Starting tomorrow afternoon, I will be available to help you in resolving this issue if you need.

**Comment by fpvandoorn** [comment]
I deleted `Dual_private` (you can do that yourself by `git rm BonnAnalysis/Dual_private.lean` and committing) and I merged master.
You can check out a "clean" copy of the repository by doing something like `git checkout master` and `git reset upstream/master --hard` (assuming you have no work on `master` yourself`).

If you've lost your own copy of `Dual_private.lean` and want it back, you can do something like 
```
git checkout complexinter_pr -- BonnAnalysis/Dual_private.lean && git restore --staged BonnAnalysis/Dual_private.lean
```

**Pull request merged**